### PR TITLE
feat(requirements): switch from simplejson to json

### PR DESCRIPTION
**Pull request by ranrib**

### CODE REVIEW

File: tests/test_trace.py [hunk 1]
```diff
@@ -524,6 +533,20 @@ def test_send_traces_sanity(wrapped_post):
...
+@mock.patch('urllib3.PoolManager.request')
+def test_send_traces_unicode(wrapped_post):
+    trace = trace_factory.get_or_create_trace()
+    runner = UnicodeReturnValueEventMock()
+    trace.set_runner(runner)
+    trace_factory.send_traces()
+    wrapped_post.assert_called_with(
+        'POST',
+        'collector',
+        body=json.dumps(trace.to_dict(), ensure_ascii=False),
```
Comment (adavidai) [inline tests/test_trace.py:545]: What are you trying to test here?
Can we avoid using the same way as in the code to generate the trace body?

File: .travis.yml [hunk 2]
```diff
@@ -10,7 +10,6 @@ before_install:
 
 install:
   - pip install -r requirements-dev.txt
-  - pip install pylint
```
[the remark this answers is not in the document]
Comment (ranrib) [reply .travis.yml:13]: Why not? pylint is a dev requirement, so there's no reason to run another command to install just it

File: epsagon/runners/python_function.py [hunk 3]
```diff
@@ -68,7 +68,7 @@ def add_json_field(self, name, data):
             return
 
         try:
-            json.dumps(data, cls=TraceEncoder, encoding='latin1')
+            json.dumps(data, cls=TraceEncoder, ensure_ascii=False)
```
[the remark this answers is not in the document]
Comment (ranrib) [reply epsagon/runners/python_function.py:71]: It's not needed, and won't help - this flag just helps us preserve any non ascii data

File: tests/test_trace.py (hunk 1 above)
Comment (ranrib) [reply tests/test_trace.py:545]: Verify that a unicode-containing trace generates the call, with all data (specifically the unicode)

File: .travis.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (adavidai) [reply .travis.yml:13]: For some reason I thought you removed it from dev requirements - never mind :)

**Review by adavidai — APPROVED** [review]

**Pull request merged**